libs/s25main/mapGenerator/Resources.cpp

```diff
@@ -60,31 +60,31 @@ namespace rttr { namespace mapGenerator {
 
     void AddObjects(Map& map, RandomUtility& rnd)
     {
-        std::set<MapPoint, MapPointLess> forbiddenArea;
+        std::set<MapPoint, MapPointLess> excludedArea;
         NodeMapBase<unsigned> probabilities;
         probabilities.Resize(map.size, 0u);
 
         // Do not allow to place trees/stone piles nearby head quarters or
         // harbor positions to avoid inaccessible harbors or invalid player positions.
 
-        auto isForbidden = [&map](const MapPoint& pt) {
+        auto harberOrHeadQuarter = [&map](const MapPoint& pt) {
```

Comment:
harberOrHeadQuarter -> harborOrHeadquarter

Reply:
BTW: Maybe check the other code(s) too while you are at it: It is either HQ or Headquarter. I'm not native english but I think a HeadQuarter i.e. head quarter would be a side of a coin or a sliced up head ;D (and yes I had to google this too)